cypress/support/commands.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
```

Comment:
Everything is commented out in this file. Do we really want to add it if its not being used yet?

Reply:
This is a file generated by cypress, I'm ok with removing it since we probably don't need it.